src/encoded/static/components/antibody.js

```diff
@@ -347,17 +347,19 @@ const AntibodyStatus = (props) => {
                             {Object.keys(organisms).map((organism, i) => {
                                 const terms = Object.keys(organisms[organism]);
                                 return (
-                                    <div key={i} className="antibody-status__group">
-                                        <div className="antibody-status__organism">
-                                            {organism}
+                                    <>
+                                        <div key={i} className="antibody-status__group">
+                                            <div className="antibody-status__organism">
+                                                {organism}
+                                            </div>
+                                            <div className="antibody-status__terms">
+                                                {terms.length === 1 && terms[0] === 'not specified' ? '' : terms.join(', ')}
+                                            </div>
+                                            <div className="antibody-status__status">
+                                                <Status item={status} inline />
+                                            </div>
                                         </div>
-                                        <div className="antibody-status__terms">
-                                            {terms.length === 1 && terms[0] === 'not specified' ? '' : terms.join(', ')}
-                                        </div>
-                                        <div className="antibody-status__status">
-                                            {i === 0 ? <Status item={status} inline /> : null}
-                                        </div>
-                                    </div>
+                                    </>
```

Comment:
Oh just one more and it’s good. Since the antibody-status__group div is the only component within the `<>` and `</>`, we don’t need the `<>` and `</>` at all.

Reply:
Thanks. Maybe this was a hangout from a previous change. I removed the react fragment.